hw/ip/csrng/data/csrng.hjson

```diff
@@ -20,7 +20,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_csrng",
-  version:            "2.0.0",
+  version:            "3.0.0",
```

Comment:
I believe you picked the wrong IP block here. This is CSRNG, not ENTROPY_SRC :-)